tests/resources/test-types.json

```diff
@@ -1096,7 +1123,73 @@
             "linkedTestCode": null
           }
         ]
-      }
+      },
+    {
+      "id": "37",
```

Comment:
Test-types with IDs 37, 38 and 39 already exist in the json file. I think these IDs have to be unique in the file.

Reply:
I checked and saw that the IDs are repeated so that's why I used them.